releases/release-1.30/release-notes/release-notes-draft.json

```diff
@@ -3916,5 +4604,857 @@
+    "markdown": "OpenAPI V2 will no longer publish aggregated apiserver OpenAPI for group-versions not matching the APIService specified group version. ([#123570](https://github.com/kubernetes/kubernetes/pull/123570), [@Jefftree](https://github.com/Jefftree))",
+    "author": "Jefftree",
+    "author_url": "https://github.com/Jefftree",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123570",
+    "pr_number": 123570,
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "api-machinery"
+    ]
+  },
+  "123575": {
+    "commit": "e4a14fe0f5a3ffc6057ead6dd6f4dc722718d84c",
+    "text": "Graduated pod scheduling gates to general availability.\nThe `PodSchedulingReadiness` feature gate no longer has any effect, and the\n`.spec.schedulingGates` field is always available within the Pod and PodTemplate APIs.",
+    "markdown": "Graduated pod scheduling gates to general availability.\n  The `PodSchedulingReadiness` feature gate no longer has any effect, and the\n  `.spec.schedulingGates` field is always available within the Pod and PodTemplate APIs. ([#123575](https://github.com/kubernetes/kubernetes/pull/123575), [@Huang-Wei](https://github.com/Huang-Wei)) [SIG API Machinery, Apps, Node, Scheduling and Testing]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/3521-pod-scheduling-readiness",
+        "type": "KEP"
+      }
+    ],
+    "author": "Huang-Wei",
+    "author_url": "https://github.com/Huang-Wei",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123575",
+    "pr_number": 123575,
+    "areas": [
+      "test",
+      "code-generation"
+    ],
+    "kinds": [
+      "api-change",
+      "feature"
+    ],
+    "sigs": [
+      "scheduling",
+      "node",
+      "api-machinery",
+      "apps",
+      "testing"
+    ],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "123578": {
+    "commit": "7c11cc9cfcdc54a7ca4efdc10609bf421bfa54d4",
+    "text": "kubeadm: printed all the kubelets and nodes that need to be upgraded on \"upgrade #     plan\".",
+    "markdown": "Kubeadm: printed all the kubelets and nodes that need to be upgraded on \"upgrade #     plan\". ([#123578](https://github.com/kubernetes/kubernetes/pull/123578), [@carlory](https://github.com/carlory))",
+    "author": "carlory",
+    "author_url": "https://github.com/carlory",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/123578",
+    "pr_number": 123578,
+    "areas": [
+      "kubeadm"
+    ],
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "cluster-lifecycle"
+    ],
+    "feature": true
+  },
+  "123593": {
+    "commit": "89cbd94e68ae80414dea7c14c1694e5e378a188d",
+    "text": "It is possible to configure the IDs that the Kubelet uses to create user namespaces.\n\n\nUser namespaces support is a Beta feature now.",
```

Comment:
Should this have the multiple new lines here?

Reply:
@npolshakova Good catch. I think one new line if fine here